Sources/OpenTelemetryApi/Context/OpenTelemetryContextProvider.swift

```diff
@@ -13,60 +13,41 @@ public enum OpenTelemetryContextKeys: String {
 
 public struct OpenTelemetryContextProvider {
   package var contextManager: ContextManager
+}
 
-  /// Returns the Span from the current context
-  public var activeSpan: Span? {
-    return contextManager.getCurrentContextValue(forKey: .span) as? Span
-  }
-
-  /// Returns the Baggage from the current context
-  public var activeBaggage: Baggage? {
-    return contextManager.getCurrentContextValue(forKey: OpenTelemetryContextKeys.baggage) as? Baggage
-  }
-
-  /// Sets the span as the activeSpan for the current context
-  /// - Parameter span: the Span to be set to the current context
-  public func setActiveSpan(_ span: Span) {
-    contextManager.setCurrentContextValue(forKey: OpenTelemetryContextKeys.span, value: span)
-  }
-
-  /// Sets the span as the activeSpan for the current context
-  /// - Parameter baggage: the Correlation Context to be set to the current context
-  public func setActiveBaggage(_ baggage: Baggage) {
-    contextManager.setCurrentContextValue(forKey: OpenTelemetryContextKeys.baggage, value: baggage)
-  }
+extension OpenTelemetry {
```

Comment:
the code added here is from the OpenTelemetry singleton setup, not related to the code removed